### PR TITLE
deb for syncthing

### PR DESCRIPTION
**Pull request by xpamych**

### CODE REVIEW

File: syncthing/lure.sh [hunk 1]
```diff
@@ -8,9 +8,12 @@ license=('MPL-2.0')
 provides=('syncthing')
 conflicts=('syncthing')
 deps_amd64=('glibc' 'strelaysrv')
+deps_amd64_debian=('libc6')
 build_deps=('git' 'go')
+build_deps_debian=('git' 'golang')
+
 sources_amd64=("https://github.com/${name}/${name}/releases/download/v${version}/${name}-linux-amd64-v${version}.tar.gz")
-checksums_amd64=('f91066b8588c02b5ff77aa801cc4e0319f3467325bdd75326aebccffb6495698')
+checksums=('f91066b8588c02b5ff77aa801cc4e0319f3467325bdd75326aebccffb6495698')
```
Comment (Arsen6331) [inline syncthing/lure.sh:16]: Please change this to `checksums_amd64`

```suggestion
checksums_amd64=('f91066b8588c02b5ff77aa801cc4e0319f3467325bdd75326aebccffb6495698')
```

**Review by Arsen6331 — CHANGES_REQUESTED** [review]

**Comment by Arsen6331** [comment]
Also, I've changed some things in this repo, so if you could merge those into your fork, that would be nice.

**Comment by xpamych** [comment]
In fact, I planned to add the rest of the architectures later, they will have their own sources. Therefore, amd64 was explicitly specified so that the installation would not run on other architectures.

**Comment by Arsen6331** [comment]
> In fact, I planned to add the rest of the architectures later, they will have their own sources. Therefore, amd64 was explicitly specified so that the installation would not run on other architectures.

Yeah, but the `checksums` should have the same architecture as the `sources`, since they will be different for other architectures

**Comment by xpamych** [comment]
i fix deps

**Review by Arsen6331 — APPROVED** [review]

**Comment by Arsen6331** [comment]
Thanks

**Pull request merged**